_cite/requirements.txt

```
@@ -1,6 +1,7 @@
-manubot==0.5.5
-PyYAML==6.0
-diskcache==5.4.0
-rich==12.6.0
-python-dotenv==0.21.0
-google-search-results==2.4.1
+manubot==0.*
+PyYAML==6.*
+diskcache==5.*
+rich==12.*
+python-dotenv==0.*
+google-search-results==2.*
```

I apologize if you already know this and have a good reason for specifying them as you have, but anyway...

What you have here will typically select the most recent minor version, but I could imagine a few reasons for keeping the explicit minor version around:
- minor version changes can introduce new functions which you or another package you're using might rely on, and adding a new package with older dependencies could cause your existing versions to slip back to a prior minor release
- it's helpful to know from looking at the file that you at least got it to work at the versions specified there

The `~=` specifies that the version can be at or after the least significant field, i.e. `~=1.1.5` would allow anything `>= 1.1.5` but `<1.2.0` (i.e. all patches >= current), and `~=1.1` would allow anything `>=1.1` but `<2.0` (i.e. all minor versions >= current).

```suggestion
manubot~=0.5
PyYAML~=6.0
diskcache~=5.4
rich~=12.6
python-dotenv~=0.21
google-search-results~=2.4
```

---

Yeah good idea. I'm not sure if I'm using any functions from any of the previous minor versions, but it's better to be safe. I just wanted to make sure the user is always on the latest non-major version without having to update, and this will satisfy that.

I'm also going to take to opportunity to update to the latest versions of all of them, and test that the citation process still works.

---

Just tested greenelab.com locally with the new packages, both from scratch and with cached. Various orcid and id types (300+ citations), so probably the best stress test.